src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -953,28 +953,36 @@ class AssetsRelationTest extends FlatSpec with Matchers with ParallelTestExecuti
         .save()
 
       val id = writeClient.assets.retrieveByExternalId(externalId).unsafeRunSync().id
+      println(externalId)
+      println(source)
+      println(id)
```

Comment:
I have a suspicion that https://github.com/cognitedata/cdp-spark-datasource/blob/master/src/test/scala/cognite/spark/v1/AssetsRelationTest.scala#L29 load around there may interfere with tests retries where load happens once and retries won't help refresh that view. Don't have the full picture, but removing that from per-class to per-test (and with randomized name?) might be worth a shot

Reply:
reverted that change now, wonder if theses tests that copy publicdata would suffer if publicdata changes asset hierarchies..

Reply:
seems the tests worked better when disabling parallell execution of AssetRelationTest .. maybe in conflict with itself..